spec/integration/kibana/spec.yml

```diff
@@ -125,7 +125,17 @@ spec:
     name: "tags.yml"
     required: false
     $ref: "./tags.spec.yml"
-
+  - description: Folder containing Kibana SLO assets
+      type: folder
+      name: slo
```

Comment:
@jsoriano I have a draft PR here that adds the SLO assets to the kibana spec. I plan to add some unit tests in the PR, but here's what CI is complaining about at the moment:

`could not load spec for "integration/kibana/spec.yml": could not parse folder specification file: yaml`

I have a couple of questions:
- Could you further assist why I am getting this error? I gave the name `slo` to the new folder. Shall I specify somewhere else what is considered a valid folder name? 
- Where shall I write my unit tests?
- I was reading the [Contributing guide](https://github.com/elastic/package-spec/blob/main/CONTRIBUTING.md). Shall I create a [new Change Proposal issue](https://github.com/elastic/package-spec/issues/new/choose)?

Reply:
> CI is complaining

 It turns out it was an indentation issue, which was fixed in this [commit](https://github.com/elastic/package-spec/pull/767/commits/582c5c6970cfe5a9cdf5527816e477fdfccff027)

> Where shall I write my unit tests?

I added following slo examples:
- `test/packages/good/kibana/slo/good-slo-abc-1.json`
- `test/packages/good_v2/kibana/slo/good_v2-slo-abc-1.json`
- `test/packages/good_v3/kibana/slo/good_v3-slo-abc-1.json`

> Shall I create a [new Change Proposal issue](https://github.com/elastic/package-spec/issues/new/choose)?

I didn't create any new issue, cause I found [this one](https://github.com/elastic/package-spec/issues/435).

Reply:
@mgiota - Do we need to load the SLO's inside the Kibana directory? Are there any guideline on defining SLO's.

Reply:
@muthu-mps I added it here as per @jsoriano 's recommendation in the past. Here's the [public SLO API](https://docs.elastic.co/api-reference/observability/createsloop) and the [SLO Elastic documentation ](https://www.elastic.co/guide/en/observability/current/slo-create.html)

In this PR I added some examples of SLO creation.

Reply:
In this [commit](https://github.com/elastic/package-spec/pull/767/commits/d78edf99120d88f117ce067e7b560e131fd7f053) I updated the README file to include the SLO assets. As you can see there are many other Kibana assets in the list as for example rules, ml modules, dashboards and I guess it makes sense to add `slo` here as well.